modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase

```diff
@@ -118,8 +118,6 @@ definition {
 	@description = "This is a test for LPS-98077. As a Developer, I want to the ability to nest fields"
 	@priority = "5"
 	test DuplicateANestedFieldGroup {
```

Comment:
Hey @brunoquei2105 , why are you removing `property portal.acceptance = "false";` from the test **DuplicateANestedFieldGroup** ?